File: Searching algorithms/linear.py
def linear(arr,n):
  for i in range(len(arr)):
    if arr[i]==n: return i
    else: continue
  return False

arr = [1,2,3,4,5,6,7,8,9,13,15]
n=int(input())
print(linear(arr,n))

File: Searching algorithms/linear.js
function linear(arr,n){
  for(i=0;i<=arr.length-1;i++){
    if(arr[i]==n){
      return i
    }
    else{
      continue
    }
  }
}
var arr=[1,5,6,8,12,16,17,26,44,70];
var n=8;
console.log("Your element is: "+ linear(arr,n));
